src/__tests__/new.test.ts

```diff
@@ -66,6 +66,52 @@ test('projen new --from external', () => {
   });
 });
 
+test('projen new --from external tarball', () => {
+  withProjectDir(projectdir => {
+    const shell = (command: string) => execSync(command, { cwd: projectdir });
+    // downloads cdk-appsync-project-1.1.3.tgz
+    shell('npm pack cdk-appsync-project@1.1.3');
+
+    execProjenCLI(projectdir, ['new', '--from', './cdk-appsync-project-1.1.3.tgz', '--no-post', MIN_NODE_VERSION_OPTION]);
+
+    // patch the projen version in package.json to match the current version
+    // otherwise, every bump would need to update these snapshots.
+    sanitizeOutput(projectdir);
+
+    // compare generated .projenrc.js to the snapshot
+    const actual = directorySnapshot(projectdir, {
+      excludeGlobs: [
+        '.git/**',
+        '.github/**',
+        'node_modules/**',
+        'yarn.lock',
+      ],
+    });
+
+    expect(actual).toMatchSnapshot();
```

Comment:
These snapshots are pretty big and have a lot that isn't important - maybe we can reduce it to something like:
```suggestion
    expect(actual['package.json']).toMatchSnapshot();
    expect(actual['.projenrc.js']).toMatchSnapshot();
    expect(actual['./cdk-appsync-project@1.1.3']).toBeDefined(); // check our tarball wasn't deleted
```